addon/validations/factory.js

```diff
@@ -564,7 +563,7 @@ function getCPDependentKeysFor(attribute, model, validations) {
   let dependentKeys = validations.map((validation) => {
     let { options } = validation;
     let type = validation._type;
-    let Validator = type === 'function' ? BaseValidator : lookupValidator(owner, type);
+    let Validator = type === 'function' ? BaseValidator : lookupValidator(owner, type).class;
```

Comment:
Yes, but I'm sending a PR into the polyfill to fix this.  Looks like discrepancy between polyfill and implementation within Ember.

When factory does not exist, undefined should be returned.  Currently, an empty FactoryManger object is returned.

/cc @rwjblue